bin/migrate.py

```diff
@@ -0,0 +1,18 @@
+import os
+import psycopg2
+
+
+def migrate():
+    conn = psycopg2.connect(os.environ['DB_URL'])
```

Comment:
if we bind the DATABASE ... i think this variable will be DATABASE_URL

https://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html#DATABASE-URL